big-practice/src/js/dashboard.js

```diff
@@ -34,6 +35,14 @@ const showModal = () => {
   modal.classList.add('modal-show');
 };
 
+const hideBtn = (button) => {
+  button.classList.add('hide-btn');
```

Comment:
Instead of using hide-btn, that works for only button. You can use .hide for all elements